modules/setting/actions.go

```diff
@@ -10,7 +10,8 @@ import (
 // Actions settings
 var (
 	Actions = struct {
-		Storage           // how the created logs should be stored
+		Storage                   // how the created logs should be stored
```

Comment:
One more thing, the `Storage` field should have explicit name , like ` Storage Storage`, to avoid implicit accessing some fields into `Storage struct` from `Actions`

Reply:
It should be renamed `LogStorage Storage`. Because now we have another `ArtifactStorage`.